src/_P098_PWM_motor.ino

```diff
@@ -369,17 +383,41 @@ boolean Plugin_098(uint8_t function, struct EventStruct *event, String& string)
         UserVar[event->BaseVarIndex + 1] = limitA_triggered ? 1 : 0;
         UserVar[event->BaseVarIndex + 2] = limitB_triggered ? 1 : 0;
         UserVar[event->BaseVarIndex + 3] = limitApos;
+
+        success = true;
       }
       break;
     }
 
+    case PLUGIN_TIME_CHANGE:
+    {
+      P098_data_struct *P098_data =
+        static_cast<P098_data_struct *>(getPluginTaskData(event->TaskIndex));
+
+      P098_data->timeChanged();
```

Comment:
I thought the `getMicros64()` function might be affected by a system time update. As it is used to calculate the duration between two `PLUGIN_READ` calls and thus to estimate the new position, it is reset after a system time change.

Do you know if the `getMicros64()` is immune to changes in system time? Then I am happy to remove it.

Reply:
It is the equivalent of `millis()`, so only related to the uptime.
And since it takes quite a while before uint64_t overflows, it may be considered to always increment :)

Reply:
We should not underestimate the life span of some Arduinos ;-)

Ok, I will remove it :+1: